bot/cogs/LastFM.py

```diff
@@ -0,0 +1,208 @@
+        embed.add_field(name='Artist',value=data['artist']['name'])
+        embed.add_field(name='Album',value=data['album']['title'])
+        embed.set_thumbnail(url=data['album']['image'][2]['#text'])
+        await ctx.send(embed=embed)
+      except:
+        await ctx.send('Invalid search term, try again')
+
+    @commands.command(
+      name="getalbum",
+      brief="a command to get info about a album, takes in an album name or album name and artist, formatting for the second is album name, artist with a comma seperating the two"
+    )
+    async def getalbum(self,ctx,*,args):
+      #try:
+        if ', ' in args:
+          args=args.split(', ')
+          data= get_album(args[0],args[1])
+        else:
+          data = get_album(args)
+        embed=discord.Embed(title=data['album']['name']+' by '+data['album']['artist'],url=data['album']['url'])
+        embed.add_field(name='Artist',value=data['album']['artist'])
+        embed.set_thumbnail(url=data['album']['image'][2]['#text'])
+        try:
+          embed.add_field(name='Release Data',value=data['album']['wiki']['published'],inline=False)
+        except:
+          'no release date'
+        try:
+          embed.add_field(name='About',value=data['album']['wiki']['summary'].split('<a',1)[0])
+        except:
+          'no summary to send'
+        await ctx.send(embed=embed)
+      #except:
+      #  await ctx.send('Invalid search term, try again')
+      
+    @commands.command(
+      name="getartist",
+      brief="A command that get's info about an artist, takes in an artist name"
+    )
+    async def getartist(self,ctx,*,args):
+      try:
+        data = get_artist(args)
+        album_data = get_data('artist.gettopalbums&artist='+urllib.parse.quote(data['name']))['topalbums']['album']
+        bio = data['bio']['summary'].split('\n',2)[0].split('<a',1)[0]
+        if bio == '':
+          bio = data['bio']['summary'].split('\n',2)[1].split('<a',1)[0]
+        embed=discord.Embed(title=data['name'],url=data['url'],description=bio)
+        embed.set_thumbnail(url=album_data[0]['image'][2]['#text'])
+        similar = ''
+        top_albums =''
+        for f in album_data[:10]:
+          top_albums=top_albums+f['name']+'\n'
+        for f in data['similar']['artist']:
+          similar= similar+f['name']+'\n'
+        embed.add_field(name='Top Albums',value=top_albums,inline=True)
+        embed.add_field(name='Similar Artists',value=similar,inline=True)
+        await ctx.send(embed=embed)
+      except:
+        await ctx.send('Invalid search term, try again')
+
+    @commands.command(
+      name="topsongs",
+      brief="Gets a list of the top songs on the world charts, this command doesn't require any args"
+    )
+    async def topsongs(self,ctx):
+      data=top_tracks()
+      embed=discord.Embed(title='Top 10 Tracks',url='https://www.last.fm/charts')
+      count=1
+      for f in data[:10]:
```

Comment:
you can use `enumerate()` here instead of a `count` var, in order to be more pythonic